alws/routers/builds.py

```diff
@@ -35,21 +27,20 @@
 
 @router.post('/', response_model=build_schema.BuildCreateResponse)
 async def create_build(
-            build: build_schema.BuildCreate,
-            user: models.User = Depends(get_current_user),
-            db: AsyncSession = Depends(get_db)
-        ):
-    try:
-        return await build_crud.create_build(db, build, user.id)
-    except PermissionDenied:
-        return HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED,
-            detail='User has no permissions to create build'
-        )
+    build: build_schema.BuildCreate,
+    user: models.User = Depends(get_current_user),
+    db: AsyncSession = Depends(get_db),
+):
+    return await build_crud.create_build(db, build, user.id)
```

Comment:
Why `PermissionDenied` handling is removed?

Reply:
because we already have middleware that catches `PermissionDenied`
https://github.com/AlmaLinux/albs-web-server/blob/master/alws/middlewares/permissions.py#L14